Netkan/KS/KSAPI.cs

```diff
@@ -11,8 +11,10 @@ namespace CKAN.NetKAN
     /// </summary>
     internal class KSError
     {
+        #pragma warning disable 0649
```

Comment:
This disables the 'never assigned to' warning, yes? But I presume we *do* need these attributes; do we know where they *do* get assigned? (If so, we should comment that, otherwise I'll come back and be confused again in the future.)